services/uinotification/notifier.go

```diff
@@ -41,8 +41,10 @@ func NewNotifier() notify.Notifier {
 	return ns
 }
 
-func init() {
+// Init initializes notification service, we need to create queue so don't run RegisterNotifier in init
+func Init() error {
 	notify.RegisterNotifier(NewNotifier())
+	return nil
```

Comment:
Why return an error, when you never return an error?

Reply:
Done.